Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:latest
```

Comment:
I guess on a “release” image we would want to request a specific version of the base OS image.

I've read that Alpine Linux is commonly used for making lightweight containers. Maybe I could try running on Alpine at some point. However, since it is based on the [musl libc][5], some testing would be needed.

I found a list of [Docker images cached by GitHub runners][6]. There are multiple versions of alpine, debian and ubuntu.

[5]: https://musl.libc.org/
[6]: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#cached-docker-images

Reply:
Yes, that makes sense.

Using alpine as a base might require some tinkering. Glibc can be installed, but with every install, the image size grows steeply. 

Also, for usage with distrobox, ubuntu might be more convinient for users since the environment is familiar. Not only that, installation of many python packages are complicated on alpine when pacakges like Numpy or Pandas contain precompiled binaries. This will be detected as not being compaticle with musl glibc and then requires building, which fails if `build-base` is not installed. If it is installed, then the build usually takes a long time [[1]](https://pythonspeed.com/articles/alpine-docker-python/).  

If we can remove build dependencies and only retain runtime dependencies, it would save some space.

---

[1] : https://pythonspeed.com/articles/alpine-docker-python/

Reply:
Thanks for the info on Alpine! As for moving build dependencies, this would be very welcome, but it can be deferred to a subsequent pull request.